vhdbuilder/scripts/cni-prefetch-driver.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+CNI_Prefetch_PATH="/opt/azure/containers/cni-prefetch.sh"
+if [[ ! -f "$CNI_Prefetch_PATH" ]]; then
+    echo "CNI Prefetch file does not exist at: $CNI_Prefetch_PATH"
+    exit 1
+fi
+
+chmod +x $CNI_Prefetch_PATH
+echo "running cni-prefetch.sh ..."
+$CNI_Prefetch_PATH
```

Comment:
will need to test and make sure bash likes this and property treats it as a script invocation

Reply:
might need `sudo /bin/bash $CNI_PREFETCH_SCRIPT_PATH` here instead